src/olympia/addons/cron.py

```diff
@@ -38,7 +38,18 @@ def update_addon_average_daily_users():
 
     kwargs = {'id_field': 'pk'}
     if waffle.switch_is_active('use-bigquery-for-addon-adu'):
-        d = get_addons_and_average_daily_users_from_bigquery()
+        # BigQuery does not have data for add-ons with type other than those in
+        # `ADDON_TYPES_WITH_STATS` so we use download counts instead.
+        # See: https://github.com/mozilla/addons-server/issues/14609
+        d = list(
```

Comment:
Can we take the chance to make a change to have a longer variable name please?  Single letter vars can be problematic.